### PR TITLE
Convert BOOL and BOOLEAN to TINYINT(1)

### PR DESCRIPTION
**Pull request by klausgreulich**
No merge conflicts anymore and put the two datatypes in a set ...

## Summary by CodeRabbit

- **Refactor**
	- Enhanced data type translation in database migration to support both "BOOL" and "BOOLEAN" types, ensuring accurate conversion.

### CODE REVIEW

File: src/sqlite3_to_mysql/transporter.py [hunk 1]
```diff
@@ -295,7 +295,7 @@ def _translate_type_from_sqlite_to_mysql(self, column_type: str) -> str:
                 if "UNSIGNED" in self._mysql_integer_type:
                     return f"{match.group(0).upper()}{length} UNSIGNED"
                 return f"{match.group(0).upper()}{length}{' UNSIGNED' if unsigned else ''}"
-        if data_type == "BOOLEAN":
+        if data_type in {"BOOL", "BOOLEAN"}:
```
[the remark this answers is not in the document]
Comment (techouse) [reply src/sqlite3_to_mysql/transporter.py:298]: @klausgreulich can you please add an explicit test for `BOOL`?

[the remark this answers is not in the document]
Comment (klausgreulich) [reply src/sqlite3_to_mysql/transporter.py:298]: for column in sqlite_column_types + ("INT64",):

Would you like me to add "BOOL" to sqlite_column_types or add it to the tupel like "INT64"?

[the remark this answers is not in the document]
Comment (techouse) [reply src/sqlite3_to_mysql/transporter.py:298]: Yeah, sure.

[the remark this answers is not in the document]
Comment (klausgreulich) [reply src/sqlite3_to_mysql/transporter.py:298]: Hi! I added tests for BOOL where BOOLEAN is tested. Don't know why it is tested in two places, but I did it according to the tests for BOOLEAN.

**Review by techouse — APPROVED** [review]

**Comment by klausgreulich** [comment]
@techouse I saw it already yesterday. 🙂 You are very much welcome. Glad I could help a little bit!

**Pull request merged**